hvplot/converter.py

```diff
@@ -2041,7 +2042,22 @@ def labels(self, x=None, y=None, data=None):
         self.use_index = False
         data, x, y = self._process_chart_args(data, x, y, single_y=True)
 
-        text = self.kwds.get('text', [c for c in data.columns if c not in (x, y)][0])
+        text = self.kwds.get('text')
+        if not text:
+            text = [c for c in data.columns if c not in (x, y)][0]
+        text_cols = re.findall(r"\{([^{}]+)\}", text)
```

Comment:
You didn't like my suggestions to leverage `.format()` ? :D I find a bit weird having to handle ourself parsing format strings with `re` and then splitting on `:`, could it be we're missing cases? `!r` comes to mind, even if it doesn't look useful in this case someone might still want to try that.

Reply:
Thanks for the insight and insisting; I didn't understand how it was relevant initially, but now it's much more elegant!

Reply:
Perfect! Thanks a lot for bearing with me :D